Add accessibility statement

pezholio opened the pull request:
This adds an accessibility statement to the service, and links to it from the footer:

![image](https://user-images.githubusercontent.com/109774/65602580-05fefd00-df9c-11e9-8d2d-a229e817311a.png)

lawrence-dxw commented:
The accessibility statement link from the front page (`http://localhost:3100` in development) redirects back to the front page, when not logged in. Probably something needs to be added to a whitelist somewhere.

pezholio commented:
Yikes, good catch. I'll get on that

pezholio commented:
@lawrence-dxw I've moved the page to its own controller, skipped the `ensure_user_logged_in` before action and added a test. All works for non-logged in users now
